README.md

```diff
@@ -37,6 +37,9 @@ We maintain documentation in this repo:
     * **[@dotcom-reliability-kit/middleware-render-error-info](./packages/middleware-render-error-info/#readme):**<br/>
       Express middleware to render error information in a browser in a way that makes local debugging easier
 
+    * **[@dotcom-reliability-kit/opentelemetry](./packages/opentelemetry/#readme):**<br/>
+      A utility function using Opentelemtery, providing health, performance, and latency data about an FT system
```

Comment:
**suggestion:** thinking about a description for this package and I wonder if we can simplify it a bit by leaning on the OpenTelemetry docs?

```suggestion
      An [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/) client that's preconfigured for drop-in use in FT apps.
```